src/components-examples/material/select/select-reactive-form/select-reactive-form-example.html

```diff
@@ -0,0 +1,46 @@
+<form [formGroup]="form">
+  <h4>mat-select</h4>
+  <mat-form-field>
+    <mat-label>First Course</mat-label>
+    <mat-select [formControl]="firstCourseControl" name="food">
```

Comment:
Couldn't we have one set of selects that are preselected? It seems redundant to double to amount of markup just so we can show how to preselect something.